Assets/Scripts/RandomItemDrop.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) TigardHighGDC
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomItemDrop : MonoBehaviour
+{
+    // you will probably want to move things around I just have it how I learned / how I stole it from a different
+    // project

```

Comment:
Should I worry about changing anything? It all looks good to me.

Reply:
I just thought you had a way of organizing things but I didnt remember so I just used what I had you dont have to worry about changing things.

Reply:
Looks fine to me so I'm going to remove this comment.